### PR TITLE
2D : Simple mouse drawing demo

### PR DESCRIPTION
**Pull request by zvodd**

### CODE REVIEW

**Comment by pwab** [comment]
Works great for me. 👍 
I have two points to add:
- It would be nice, if the clicks generate a cirle or rect at the clicked position (the starting and ending points). This would imitate the behaviour of most vector programs.
- Could this demo be expanded with the possibility to manually draw and erase dots? This would lead to a simple paint-like program.

File: 2d/simple_mouse_drawing/MouseDrawing.gd [hunk 1]
```diff
@@ -0,0 +1,57 @@
+extends Node2D
+
+const line_width = 10
```
Comment (pwab) [inline 2d/simple_mouse_drawing/MouseDrawing.gd]: Not sure about coding conventions for gdscript, but constants should be named in CAPS.

File: 2d/simple_mouse_drawing/MouseDrawing.gd [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	
+	# basic way to keep track of the added lines
+	node.add_to_group("DrawLines")
+
+
+func create_poly_coords_for_line(start, end, width):
+	# Creates 2 points each for start and end point of a line;
+	# these points make up the corners of the rectangle that is the line representation.
+	
+	width = width / 2
```
Comment (pwab) [inline 2d/simple_mouse_drawing/MouseDrawing.gd]: Here should be a short comment, what this line means.

**Comment by aaronfranke** [comment]
Superseded by https://github.com/godotengine/godot-demo-projects/pull/89, this should be closed.

**Comment by akien-mga** [comment]
Thanks, closing as superseded by #89 then.

**Pull request closed without merging**